src/linter/messages.ts

```diff
@@ -480,8 +480,8 @@ export const MESSAGE_INFO = {
 			`Deprecated declaration of renderer ${rendererName ? `'${rendererName}' ` : ""}for Control '${className}'`,
 		details: ({className, rendererName}: {className: string; rendererName: string | undefined}) => {
 			const rendererModuleName = rendererName ? `'${rendererName.replace(/\./g, "/")}'` : "renderer";
-			return `Defining the 'renderer' for the control '${className}' as string may lead to synchronous loading ` +
-				`of the ${rendererModuleName} module. ` +
+			return `Defining the 'renderer' for control '${className}' as by its name may lead to synchronous ` +
```

Comment:
My English isn't good enough to fully judge this, but was "as by..." intended?

Reply:
Nope 🙈
Thanks 👍🏻